policies/coding-style.md

```diff
@@ -641,9 +641,32 @@ but do this instead:
     if ((p = BN_new()) == NULL)
 ```
 
-Do not place binary logical operators `&&` and `||` at the end of a line
-but put them at the beginning of the following line.<br/>
-They may get an extra indentation (+4 characters).<br/>
+If you need to break an expression into multiple lines,
+make the line break before an operator, not after.
+It is preferred that such a line break is made
+before as low priority an operator as possible.<br>
```

Comment:
Remove `<br>` and add an empty line before `Example:`

Reply:
Ah, must have been because I followed the link to a specific fixup commit, not realising that wasn't the last one.